tests/OmniSharp.Roslyn.CSharp.Tests/IntellisenseFacts.cs

```diff
@@ -259,6 +259,7 @@ public MyClass2()
 
             var completions = await FindCompletionsAsync(filename, source);
             ContainsCompletions(completions.Select(c => c.CompletionText), "Foo");
+            ContainsCompletions(completions.Select(c => c.ReturnType), "string");
```

Comment:
without the changes from this PR, this assert fails (for example on current master). It would have prevented the regression too.